.cursor/skills/handle-pr-review/SKILL.md

```
@@ -0,0 +1,116 @@
+
+**ユーザーの承認を得てから Step 3 に進む。**
+
+## Step 3: 修正の実装
+
+承認されたコメントに対して修正を行う:
+
+1. 対象ファイルを読み、指摘箇所を確認
+2. 修正を実装
+3. `bun run lint` でエラーがないか確認
+4. 全修正をまとめて 1 コミット:
+   ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)